src/org/labkey/test/components/glassLibrary/grids/ResponsiveGrid.java

```diff
@@ -87,36 +87,33 @@ public Boolean hasData()
         return elementCache().emptyGrid.isEmpty();
     }
 
-    public List<WebElement> getRows()
-    {
-        return elementCache().getRows();
-    }
-
-    // TODO Responsive grids cannot be sorted, this should be moved to QueryGrid (or what ever it will be called).
```

Comment:
Are you absolutely sure that all grids can be sorted?

Reply:
Not all can.  The ones that don't have drop-menus in the header cells and the ones that don't have omniBoxes can't be sorted.  Until we have a separate class for un-sortableGrid, or one for a SortableGrid, it doesn't hurt to support those functions here and if someone tries to call these functions on grids that can't be sorted, they'll find out quickly enough